### DISCUSSION TITLE
Independant isActive states for button groups

### DISCUSSION THREAD

**Discussion by Jersh** [element-plus/element-plus#13255] [Q&A] [answered]
```
<div class="dateNav">
   <el-button-group>
      <el-button type="danger" plain> Days </el-button>      
      <el-button type="danger" plain> Lates </el-button>      
      <el-button type="danger" plain> Nights </el-button>   
   </el-button-group>   
</div>
<div class="dtFilter">
   <el-button-group>
      <el-button type="danger" plain> Unconfirmed </el-button>      
      <el-button type="danger" plain> Confirmed </el-button>      
      <el-button type="danger" plain> All </el-button>    
   </el-button-group>
</div>
````

Given the code above is there a way to have a button from each group have the active state at the same time?

I.e I'd like to have Days selected and Unconfirmed. if I selected lates then days will become inactive in favour of Lates but Unconfirmed will stay as active.

**Comment by ryuhangyeong** [comment]
Hello!

I think what you want is `el-checkbox`. Is that correct?

https://element-plus.org/en-US/component/checkbox.html#button-style

**Reply by Jersh** [reply]
Not quite,
I have three buttons in two buttons groups, I want to be able to select a max of 1 button from each group at a time.
![image](https://github.com/element-plus/element-plus/assets/6068210/94800ca0-200a-4f30-b729-e55766bc8739)

in this rendered example I can only select one of the buttons. I'd like to be able to select 1 button from each group as like a nested option.

The issue with using checkbox-button with :max="1" is that you have to unselect the button to select the next one, I want this to behave like a normal button in that picking one should deselect the other, bit just want to be able to have two independant button groups on the same page.

**Comment by Jersh** [comment]
Whilst doing another component I've noticed in the docs  [here](https://element-plus.org/en-US/component/form.html#size-control) that there is an example of exactly what I am trying to achieve, two groups of three buttons that can be independently selected within the groups.
however when I copy the code snipped into my workspace it appears to work 
![image](https://github.com/element-plus/element-plus/assets/6068210/0571d108-b60b-4099-93e4-2517b52a7029)

Until I chose another one of the options, then I seem to loose the isActive state
![image](https://github.com/element-plus/element-plus/assets/6068210/c33b20b0-432a-4426-b0f9-bde6cdff416a)

**Comment by Jersh** [comment, marked as answer]
Fixed it, turns out el-radio-button has the behaviour I wanted.
Just need to sort the styles out now as it doesn't seem to respond to  type="danger"
